src/models/encounter/useEncounterFieldSchemas.js

```diff
@@ -15,9 +15,10 @@ export default function useSightingFieldSchemas() {
   const intl = useIntl();
   const { data, loading, error, siteSettingsVersion } =
     useSiteSettings();
-  if (loading || error) return null;
 
   const encounterFieldSchemas = useMemo(() => {
+    if (loading || error) return null;
```

Comment:
In `useOptions` instead of exiting early from useMemo, you used the pattern:
```
  const {data, loading, error} = useCustomHook();
  const formattedData = useMemo(() => {
    // logic to format the data
  }, [data]);
  
  if (loading || error) return someDefaultValue;
  return formattedData;
```

I can think of arguments for either one, but I think we should be consistent.

Reply:
hmm, i'll update useOptions. i think short-circuiting inside the hook prevents all the code below it from running an extra time.